pkgs/development/compilers/ecl/default.nix

```diff
@@ -36,6 +36,17 @@ stdenv.mkDerivation {
   };
 
   patches = [
+    # https://git.sagemath.org/sage.git/tree/build/pkgs/ecl/patches/ECL_WITH_LISP_FPE.patch?id=f82c716fdf9c6e91a07166d36b6329a15ecfb41d
```

Comment:
Can you please use fetchurl?

Reply:
Oh derp. You can use fetchurl but fetchpatch is better because it normalizes the patch that changing meta data does not cause a hash mismatch. Sorry about my derp.

Reply:
Don't worry! I thought about that (it was done like this before), but the patches are plain files committed to Sage's repository, they are not dynamically generated. Does fetchpatch still provide an advantage in this case?